plasmapy/mathematics/mathematics.py

```diff
@@ -194,32 +194,30 @@ def Fermi_integral(x, j):
     ValueError
         If the argument is not entirely finite.
 
-    See also
+    See Also
     --------
     plasma_dispersion_func
 
     Notes
     -----
-    The complete Fermi-Dirac integral is defined as: [1]_
+    The `complete Fermi-Dirac integral
+    <https://en.wikipedia.org/wiki/Complete_Fermi-Dirac_integral>`_ is
```

Comment:
Why the change in links here?

Reply:
I was hoping to have this appear as an in-line link, which makes it easier to get to compared to the references.